### PR TITLE
Modify text of welcome email

### PR DESCRIPTION
**Pull request by azimpradhan**
When a user creates an account on Lagunita or CME (or SUclass backdoor),
they are sent an activation email. Unfortunately, the email is
embarrassing and bad. This commit modifies the text of this email as
requested by course-ops

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@azimpradhan Is this ready yet for review or am I jumping the gun?

**Comment by azimpradhan** [comment]
@stvstnfrd, yes it is ready for review

File: lms/envs/devstack.py [hunk 1]
```diff
@@ -156,3 +156,7 @@ def should_show_debug_toolbar(_):
     "API_ACCESS_KEY": "BBBBBBBBBBBBBBBBBBBB",
     "API_SECRET_KEY": "CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC",
 }
+##### Zendesk #####
+ZENDESK_URL = 'https://stanfordonline.zendesk.com/hc/en-us'
```
Comment (stvstnfrd) [inline lms/envs/devstack.py]: Remove.

File: lms/envs/common.py [hunk 2]
```diff
@@ -991,7 +991,7 @@
 HELP_MODAL_LINKS = []
 
 ##### Zendesk #####
-ZENDESK_URL = None
+ZENDESK_URL = 'https://stanfordonline.zendesk.com/hc/en-us'
```
Comment (stvstnfrd) [inline lms/envs/common.py]: Remove; don't include Stanford assets in repo.

Comment (stvstnfrd) [inline lms/envs/common.py]: Move this to test.py, but change it to a bogus URL: `https://example.com/zendesk`

**Comment by stvstnfrd** [comment]
Rebase as discussed F2F.

+1

**Pull request merged**